UnityProject/Assets/Scripts/Chat/DiscordWebhookMessage.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System.Collections.Specialized;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Net;
+using System.Text.RegularExpressions;
+using DatabaseAPI;
+
+
+namespace DiscordWebhook
+{
+	public class DiscordWebhookMessage : MonoBehaviour
```

Comment:
Could you add a short class description? It'll make things easier to for future constributors to userstand.

Reply:
Maybe add a note somewhere that this won't work without setting up the webhooks on the server.

Reply:
Added a comment.